### PR TITLE
Applivery Tutorial

### PR DESCRIPTION
**Pull request by CesarTrigo**
Applivery tutorial (plus figures and screenshots) providing guidance on how to deploy apps to Applivery from Bitrise

### CODE REVIEW

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+---
+title: Deploy apps to Applivery from Bitrise
+redirect_from: "/tutorials/deploy/deploy-apps-to-applivery-from-bitrise/"
+menu:
+  deploy-main:
+    weight: 5
+
+---
+# What is Applivery?
```
Comment (zoltan-baba) [inline _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]:
```suggestion
## Introduction to Applivery
```

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+### Step 1 - Adding Applivery Deploy Step to your workflow
+Add `Applivery iOS Deploy` or `Applivery Android Deploy` step to your bitrise workflow. Make sure you add the step after the app build process.
+
+![Applivery Workflow Step](/img/tutorials/deploy/applivery/tutorial1.png)
+
+### Step 2 - Getting your Applivery App Token
+Now it’s time to link your Bitrise App with your Applivery App. To do so, you’ll need your **Applivery App Token**. ([Read more about how to get your App Token](https://www.applivery.com/docs/rest-api/authentication/))
+
+### Step 3 - Setting up Applivery App Token
```
Comment (zoltan-baba) [inline _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]:
```suggestion
3. Open your app on Bitrise and click the **Workflows** tab to open the Workflow Editor. 
4. Go to the **Secrets** tab. 
5. Click **Add New** and type `APPLIVERY_APP_TOKEN` in the key input field. 
6. Paste your Applivery App Token in the value input field and click **Save**.
```

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+Go to Bitrise  dashboard top menu and select **Secrets** menu option. Click **Add New** button and type `APPLIVERY_APP_TOKEN` in the left side input form. Then paste your **Applivery App Token** captured in the previous step in the right input field and click **Save**.
```
Comment (zoltan-baba) [inline _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]:
```suggestion
```

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+![Configuring Applivery App Token](/img/tutorials/deploy/applivery/tutorial2.png)
+
+### Step 4 - Configuring the Applivery step
```
Comment (zoltan-baba) [inline _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]:
```suggestion
### Configuring the Applivery Step
```

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+| App Token | `String` | Applivery App token. By default gets `$APPLIVERY_APP_TOKEN` secret var. |
+| Changelog | `String` | Additional build/release notes or changelog attached to the deploy. |
+| Notify Collaborators? | `Boolean` | Automatically notify your project Collaborators vía emai.|
+| Notify Employees? | `Boolean` | Automatically notify your project Employees vía emai. |
+| Notification message | `String` | Notification message to be sent along with the email notification. |
+| Tags | `String` | Comma-separated list of tags to easily identify the build or multitrack App Distribution |
+| Version name | `String` | Human readable version name for a better identification of the build. |
+| Upload Certificates | `Boolean` | Download your code signing files from Bitrise Code Signing Tab and upload them to Applivery. |
+
+# How Distribution works in Applivery?
```
Comment (zoltan-baba) [inline _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]:
```suggestion
## Distribution with Applivery
```

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+
+![Distribution in Applivery](/img/tutorials/deploy/applivery/fig2.png)
+
+Applivery provides multiple different ways for app distribution from **customized App Stores** (public or private) to **Distribution Pages** (public, private, unlisted, or password-protected **shareable installation links**). It also enables **multitrack app delivery** based on the information gathered from your Bitrise workflows, such as **GitHub Branches, Tags or customized labels**.
```
Comment (zoltan-baba) [inline _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]:
```suggestion
Applivery provides multiple different ways for app distribution from customized App Stores (public or private) to Distribution Pages (public, private, unlisted, or password-protected shareable installation links). It also enables multitrack app delivery based on the information gathered from your Bitrise workflows, such as GitHub Branches, Tags or customized labels. 
```

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+---
+title: Deploy apps to Applivery from Bitrise
```
Comment (zoltan-baba) [inline _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]:
```suggestion
title: Deploying apps to Applivery
```

File: _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md (hunk 3 above)
Comment (CesarTrigo) [reply _articles/en/deploy/deploy-apps-to-applivery-from-bitrise.md]: Hi @zoltan-baba,
Don't you think that users should know clearly where they have to go in Bitrise dashboard?

**Comment by CesarTrigo** [comment]
Hi @zoltan-baba 
Thank you for your suggestions and fixes. I just reviewed all of them and they look good.
I believe we can proceed.
Regards,
C.

**Comment by zoltan-baba** [comment]
Thank you @CesarTrigo! Great guide, we really appreciate it - such content is always welcome!

**Pull request merged**